node/cmd/guardiand/query.go

```diff
@@ -2,34 +2,69 @@ package guardiand
 
 import (
 	"context"
+	"encoding/hex"
 	"fmt"
 	"strings"
+	"time"
 
 	"github.com/certusone/wormhole/node/pkg/common"
 	gossipv1 "github.com/certusone/wormhole/node/pkg/proto/gossip/v1"
+	"github.com/wormhole-foundation/wormhole/sdk/vaa"
+
 	ethCommon "github.com/ethereum/go-ethereum/common"
 	ethCrypto "github.com/ethereum/go-ethereum/crypto"
-	"github.com/wormhole-foundation/wormhole/sdk/vaa"
+
 	"go.uber.org/zap"
 	"google.golang.org/protobuf/proto"
 )
 
-// Multiplex observation requests to the appropriate chain
+const (
+	// requestTimeout indicates how long before a request is considered to have timed out.
+	requestTimeout = 1 * time.Minute
+
+	// retryInterval specifies how long we will wait between retry intervals. This is the interval of our ticker.
+	retryInterval = 10 * time.Second
```

Comment:
We should consider moving the ctx timeout in the watcher for the batch call here so that we can easily spot check that it is definitively less than the retryInterval, otherwise requests could pile up.